nservicebus/sqlserver/concurrency.md

```diff
@@ -26,3 +26,13 @@ In order to address the problem with excess number of unused polling threads, in
  * if last receive operation yielded no message, it increments the *consecutive failures* counter and resets the *consecutive successes* counter
  * if *consecutive successes* counter goes over a certain threshold and there is less polling threads than `MaximumConcurrencyLevel`, it starts a new polling thread and resets the *consecutive successes* counter
  * if *consecutive failures* counter goes over a certain threshold and there is more than one polling thread it kills one of the polling threads
+
+### 3.0
+
+In version 3.0 the adaptive concurrency model has been removed. Instead for each input queue SqlTransport maintains a dedicated monitoring thread. It is responsible for detecting the number of messages waiting for delivery and creating receive [tasks](https://msdn.microsoft.com/en-us/library/system.threading.tasks.task.aspx) - one for each pending message. 
```

Comment:
i think invert the order of these version? ie doesnt it make sense to have the current on first?